lib/sentry.ex

```diff
@@ -108,14 +108,14 @@ defmodule Sentry do
 
   ## Event Callbacks
 
-  You can configure the `:before_send_event` and `:after_send_event` options to
-  customize what happens before and/or after sending an event. The `:before_send_event`
+  You can configure the `:before_send` and `:after_send_event` options to
+  customize what happens before and/or after sending an event. The `:before_send`
   callback must be of type `t:before_send_event_callback/0` and the `:after_send_event`
   callback must be of type `t:after_send_event_callback/0`. For example, you
   can set:
 
       config :sentry,
-        before_send_event: {MyModule, :before_send},
+        before_send: {MyModule, :before_send},
         after_send_event: {MyModule, :after_send}
```

Comment:
Should we also rename this then? (It doesn't exist in other SDKs)
`after_send_event`

Reply:
@HazAT it doesn't exist in other SDKs as you said, and that's exactly why I left it untouched. It's one less change folks have to deal with. Honestly, I’m not even sure it's very much needed but hey, it's there 🤷 🙃